cmd/pool/order.go

```diff
@@ -100,6 +100,11 @@ var baseBidFlags = []cli.Flag{
 			"from our account into the channel; can be " +
 			"used to create up to 50/50 balanced channels",
 	},
+	cli.BoolFlag{
+		Name: "private_channel",
```

Comment:
Carry over comment here: re the name and if asks should be able to specify if they want to accept such a channel or not.

Reply:
Added, see comment above :+1: